src/Servers/IIS/AspNetCoreModuleV2/AspNetCore/aspnetcore_schema_v2.xml

```diff
@@ -0,0 +1,47 @@
+<!--
```

Comment:
Is this addition intentional? Is this different from the other aspnetcore schema file?